conf/config.go

```diff
@@ -69,11 +69,34 @@ const (
 // if configuration file found and loaded, state doesn't changed
 var state configStatus = configGood
 
-/*
-   This is the private helper function that sets up viper. This function is
-   called by the init() function only once during initialization of the package.
-*/
-func setup(locations ...string) (config, configStatus) {
+// init: First thing to run when this package is loaded by the binary.
+// Even if multiple packages import conf, this will be called and ran ONLY once.
+func init() {
+	apiConfigPath := os.Getenv("BCDA_API_CONFIG_PATH")
+	workerConfigPath := os.Getenv("BCDA_WORKER_CONFIG_PATH")
+
+	env := os.Getenv("DEPLOYMENT_TARGET")
```

Comment:
Right now, this variable is [coming from S3](https://github.com/CMSgov/bcda-ops/blob/master/terraform/modules/asg/templates/user-data.tpl#L31).  Before we remove the env variables from there, we will need to set the env variable elsewhere, possibly in the docker file?